PRODUCTIONISATION.md

```diff
@@ -97,6 +97,7 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 * [ ] 7.2 Alerts sent to relevant instant chat channel
 * [ ] 7.3 Alerts sent to relevant team email address
 * [ ] 7.4 UUID of request is sent as additional user data
+* [ ] 7.5 No reported unhandled exceptions in the past week
```

Comment:
This is ambiguous.

Are we saying:
- No exceptions in Sentry that haven't been addressed within a week
- No exceptions in Sentry for at least a week

Reply:
Updated with alternative wording, and added additional justification for the point.